web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -638,6 +642,12 @@
   })();
 
   window.liftJQuery = {
+    onEvent: function(elementOrId, eventName, fn) {
+      if (typeof elementOrId == 'string')
+        elementOrId = '#' + elementOrId;
+
```

Comment:
Haha it doesn't make me super-nervous but I can totally understand the nervousness; I'll fix it :)